x/timerstore/client/cli/query_store_keys.go

```diff
@@ -14,7 +14,7 @@ var _ = strconv.Itoa(0)
 func CmdStoreKeys() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "store-keys",
-		Short: "Query for all timer store keys",
+		Short: "Query all timer store keys",
```

Comment:
and prefixes

Reply:
Fixed in 1c3e89403e64df0743f3d8ea1cf3d33c0bfd5627